projects/packages/jitm/package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"private": true,
+	"description": "Display Just In Time Messages (JITMs) on admin pages.",
+	"homepage": "https://jetpack.com",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build": "yarn install-if-deps-outdated && yarn clean && yarn build-js",
+		"build-js": "webpack --config ./webpack.config.js",
+		"build-production": "yarn distclean && yarn install --production=false && yarn build-production-js",
+		"build-production-js": "NODE_ENV=production BABEL_ENV=production yarn build-js && yarn validate-es5 ./assets/",
+		"clean": "true",
+		"distclean": "rm -rf node_modules && yarn clean",
+		"install-if-deps-outdated": "yarn install --check-files --production=false --frozen-lockfile",
+		"validate-es5": "eslint --parser-options=ecmaVersion:5 --no-eslintrc --no-ignore"
+	},
+	"devDependencies": {
+		"eslint": "7.18.0",
+		"webpack": "4.46.0",
+		"webpack-cli": "4.5.0"
+	},
+	"engines": {
+		"node": "^12.18.0",
```

Comment:
Can we bump this to `12.20.1` now that the monorepo requires this version of node (see `.nvmrc`)?

Reply:
Fixed in commit 803c759.